### DISCUSSION TITLE
(preset) nlo's clean composite preset

### DISCUSSION THREAD

**Discussion by dawnch0rus** [ntsc-rs/ntsc-rs#107] [Presets]
https://github.com/valadaptive/ntsc-rs/assets/154479340/b20cf092-58f0-4087-9e07-ca15358514d3

this preset replicates the slight analog artifacts of clean composite video. perfect for replicating the output of a dvd player, video game console, or other sources that use cvbs.

[ntsc-rs_cvbs6.json](https://github.com/valadaptive/ntsc-rs/files/15445236/ntsc-rs_cvbs6.json)

**Comment by Degamisu** [comment]
well done! spot on.

**Comment by meowethofthedummy38** [comment]
i just made this account and im wondering how do you add videos - photos and/or files in Disscusions? just asking for a fella

**Comment by meowethofthedummy38** [comment]
wait a minute-
<img width="427" height="512" alt="7929-steamhappycool-1864614169" src="https://github.com/user-attachments/assets/0a46223c-09ff-4ea5-93ac-472e9a99068d" />

**Comment by meowethofthedummy38** [comment]
oooooooooo

**Comment by meowethofthedummy38** [comment]
im dumb.

**Reply by RayyanMakesVHS** [reply]
ARE YOU CRAZY?!

**Reply by meowethofthedummy38** [reply]
?
